lib/screens/home/home_screen.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:apexology/screens/landing/landing_controller.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+//import 'home_controller.dart';
+
+class HomeScreen extends StatelessWidget {
```

Comment:
we are using getX so replace this line with class HomeScreen extends GetView<HomeController> {